ckanext/unhcr/helpers.py

```diff
@@ -1285,3 +1285,18 @@ def data_containers_with_members_list(include_sysadmins=True, user=None):
     fn = toolkit.get_action('data_containers_with_members_list')
     containers = fn(context, {'include_sysadmins': include_sysadmins})
     return containers
+
+
+def get_resource_data_type_fields():
+    """ Get a list of fields for "data" resource not beloging to "attachment" resources """
+    return [
+        'date_range_start', 'date_range_end', 'hxl-ated',
+        'data_collector', 'version', 'process_status',
+        'identifiability'
+    ]
```

Comment:
These lists always end up going out of sync. Let's keep our schema as the single source of truth and add a `"data_only": true` or similar key to the relevant resource fields, and then we can do:

```suggestion
    schema = scheming_get_dataset_schema("dataset")                                                                                     
    
    return [f["field_name"] for f in schema['resource_fields'] if f.get("data_only")]
```

Reply:
Considering #904 we will need to cover multiple resource schemas. I'll propose a change thinking on that @amercader